test/addons/rbd-mirror/start

```diff
@@ -230,6 +230,14 @@ def deploy_vrc_sample(cluster):
         context=cluster,
     )
 
```

Comment:
Need 2 blank lines between functions.

See the linter job:
https://github.com/RamenDR/ramen/actions/runs/10924436222/job/30323476661

```
def deploy_vgrc_sample(cluster):
^
1     E302 expected 2 blank lines, found 1
```

Reply:
Done